main.go

```diff
@@ -59,15 +58,19 @@ func main() {
 		log.Fatal("missing env var SERVER_UID")
 	}
 
-	// Initialised the database connection.
+	// Load the database credentials
 	uri, dbCreds, err := loadDBCredentials()
 	if err != nil {
 		log.Fatal(errors.AddContext(err, "failed to fetch db credentials"))
 	}
+
+	// Create a connection to the database
+	ctx, cancel := context.WithTimeout(context.Background(), database.MongoDefaultTimeout)
 	db, err := database.New(ctx, uri, dbCreds, logger)
 	if err != nil {
 		log.Fatal(errors.AddContext(err, "failed to connect to the db"))
 	}
+	cancel()
```

Comment:
Why not in line 69 + defer?

Reply:
I overwrite `ctx` and `cancel` to get a context for closing out the database 🤷‍♂️  updated variable names and deferred it